projects/common/src/utilities/formatters/string/highlight.pipe.ts

```diff
@@ -2,6 +2,7 @@ import { Pipe, PipeTransform } from '@angular/core';
 import { isArray } from 'lodash-es';
 import { assertUnreachable } from '../../lang/lang-utils';
 
+// TODO: Currently htHighlight does not accommodate special characters.
```

Comment:
this is the wrong description. it does not escape reserved regex characters is the issue.

Reply:
updated